proposed/event-dispatcher.md

```diff
@@ -56,6 +56,8 @@ A Listener MAY delegate actions to other code.  That includes a Listener being a
 
 A Listener MAY enqueue information from the Event for later processing by a secondary process, using cron, a queue server, or similar techniques.  It MAY serialize the Event object itself to do so, however, care should be taken that not all Event objects may be safely serializable.
 
+A secondary process MUST assume that any changes it makes to an Event object will NOT propagate to other Listeners.  Additionally, the `isPropagationStopped()` method will not be called in this case and thus the Event is effectively unstoppable.
```

Comment:
What is implied by a "secondary process" here? I'm missing context, and don't quite understand what this paragraph is specifying. Are we talking a queue runner that has received a deferred event and listener here? If so, this needs to be part of the previous paragraph to ensure that context is retained.

Reply:
Per the previous paragraph, "Secondary process" means cron, a queue server, or "similar techniques" (ie, "something like that").  So whatever mechanism is being used for "later", it loses its bidirectional communication mechanisms.

Reply:
Then, as I noted, I think the two paragraphs need to be combined. Each of the previous paragraphs is detailing a single scenario, so when I read this one, I assumed it was separate from the previous paragraph, and thus I had questions about context. Combining it with the previous paragraph would make this eminently clear.